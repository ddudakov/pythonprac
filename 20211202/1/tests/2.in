class C(metaclass=dump):
    def __init__(self, val):
        self.val = val

    def add(self, other, another=None, end=0):
        return self.val + other + (another or self.val) + end

c = C(10)
print(c.add(9))
print(c.add(9,another=8,end=1))
