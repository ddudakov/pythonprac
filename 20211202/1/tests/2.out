__init__: (10,), {}
add: (9,), {}
29
add: (9,), {'another': 8, 'end': 1}
28
